docs/deployment/managers.md

```diff
@@ -1,7 +1,63 @@
 # Platform managers
 
-!!! tip "Under construction"
+## Rules engine
+
+Allows users to execute pre-defined actions based on the data and events in the platform.
+
+These rules are accessible in Settings > Customization > Rules engine
+
+The rules engine is designed to help users automate and streamline their cyber threat intelligence processes.
+
+## History manager
+
+This manager keeps tracks of user/connector interactions on entities in the platform.
+It is designed to help users audit and understand the evolution of their CTI data
+
+## Activity manager
+
+The activity manager in OpenCTI is a component that monitors and logs the user actions in the platform such as login, setting update and group update.
+
+## Background task manager
+
+Is a component that handles the execution of tasks, such as importing data, exporting data and mass operations
+
+## Expiration scheduler
+
+The expiration scheduler is responsible for monitoring expired elements in the platform.
+It cancels the access rights of expired user accounts and revokes expired indicators from the platform
+
+## Synchronization manager
+
+The synchronization manager enables the data sharing between multiple OpenCTI platforms. It allows the user to create and configure synchronizers which are processes that connect to the live streams of remote OpenCTI platforms and import the data in to the local platform. 
+
+## Retention manager
+
+The retention manager is a component that allows the user to define rules to help delete data in OpenCTI that is no longer relevant or useful. This helps to optimize the performance and storage of the OpenCTI platform and ensures the quality and accuracy of the data.
+
+## Notification manager
+
+The notification manager is a component that allows the user to customize and receive alerts about events/changes in the platform.
+
+!! Is the next paragraph needed here?
```

Comment:
I think you can keep it here

Reply:
I recommend redirecting to this page rather than noting it here: https://docs.opencti.io/latest/usage/notifications/